fe/src/pages/ProductPage/ProductPage.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Flex,
+  Box,
+  Input,
+  Text,
+  FormLabel,
+  Textarea,
+  Button,
+} from '@chakra-ui/react';
+import {
+  Formik,
+  Form,
+} from 'formik';
+
+interface MyFormValues {
+  name: string;
+  description: string;
+}
+
+function ProductPage(): JSX.Element {
+  const initialValues: MyFormValues = { name: '', description: '' };
+  return (
+    <Flex direction="column" align="center">
+      <Text fontSize="30px" mb="30px">Product</Text>
+      <Box align="center">
+        <Formik
+          initialValues={initialValues}
+          onSubmit={values => {
+            console.log({ values });
```

Comment:
Also for now you will have to add `// eslint-disable-next-line` before this console.log line, otherwise the CI will fail and this could not be merged.